src/js/common/components/form-elements/ErrorableRadioButtons.jsx

```diff
@@ -127,10 +130,22 @@ class ErrorableRadioButtons extends React.Component {
       return output;
     });
 
+    const fieldsetClass = classNames({
+      'fieldset-input': true,
+      'usa-input-error': this.props.errorMessage,
+      [`${this.props.additionalFieldsetClass}`]: !!this.props.additionalFieldsetClass
+    });
```

Comment:
Ah, new JS features are awesome. Does this have to be written wrapped in a template string, or can it be `[this.props.additionalFieldsetClass]`?

Reply:
Good catch, @ncksllvn, this has been simplified.